src/components/Comment/Comment.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div class="unnnic-comment">
+    <img class="unnnic-comment__avatar" :src="avatar" />
+    <div class="unnnic-comment__content">
+      <div class="unnnic-comment__content__header">
+        <div class="unnnic-comment__content__header__title">{{ title }}</div>
+        <div class="unnnic-comment__content__header__time">{{ time }}</div>
+        <div class="unnnic-comment__content__header__actions">
+          <slot name="actions"></slot>
+        </div>
+      </div>
+      <div class="unnnic-comment__content__text">{{ text }}</div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'unnnic-comment',
+  components: {},
+  props: {
+    avatar: {
+      type: String,
+      default: null,
+    },
+    title: {
+      type: String,
+      default: null,
+    },
+    time: {
+      type: String,
+      default: null,
+    },
+    text: {
+      type: String,
+      default: null,
+    },
+  },
+};
+</script>
+
+<style scoped lang="scss">
+@import '../../assets/scss/unnnic.scss';
+
+.unnnic-comment {
+  display: flex;
+
+  &__avatar {
+    display: flex;
+    align-self: flex-start;
+    width: 56px;
```

Comment:
Shouldn't it be in the ds? There are different types of avatar size.

Reply:
this in itself is outside the design system, it is what was passed in the handoff